app/packages/core/src/components/ColorModal/utils.ts

```diff
@@ -69,13 +69,30 @@ export const validateJSONSetting = (
   });
 };
 
+export const validateLabelTags = (
+  obj: ColorSchemeInput["labelTags"]
+): ColorSchemeInput["labelTags"] => {
+  if (typeof obj === "object" && obj !== null) {
```

Comment:
if the type of obj is "object" it can't be null no? meaning you can remove `obj !== null`